admin-app/src/constants/colors.ts

```diff
@@ -39,6 +40,7 @@ export enum ColorsLight {
 	chatHoverBackgroundColor = "#C8C8C8",
 	inputCheckedColor = "#17A625",
 	inputUncheckedColor = "#BDBDBD",
+	breadcrumbsBorderColorActive = "#ff0000ff",
```

Comment:
Are you sure the colors have to be the same?

Reply:
Yes